XcodeAutoBasher/XcodeAutoBasher/VOKXcodeAutoBasher.m

```diff
@@ -69,6 +74,11 @@ - (id)initWithBundle:(NSBundle *)plugin
     return self;
 }
 
+- (void)applicationDidFinishLaunching:(NSNotification *)notification
+{
+    [self setupMenuItem];
```

Comment:
I don't see this being removed from wherever it used to be—is there any risk of it being called twice and yielding duplicate menu items?